.github/workflows/ios_android.yml

```diff
@@ -22,6 +26,11 @@ jobs:
         uses: jwlawson/actions-setup-cmake@v1.8
         with:
           cmake-version: '3.19.6' # See https://gitlab.kitware.com/cmake/cmake/-/issues/22021
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: ${{ inputs.java-version }}        
```

Comment:
Does the build fail without this or something? I'm trying to understand the implication... does this mean ADO build agents don't work with newer Android projects or something?

Reply:
Yes by default the build machines were pointing to JDK 8, so omitting this step was causing a build failer here is the documentation about this task https://github.com/actions/setup-java